.drone.yml

```diff
@@ -117,6 +117,23 @@ trigger:
   event:
   - push
 
+
+---
+kind: pipeline
+type: exec
+name: integration-tests-jail-unjail
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: integration-tests
+  commands:
+  - cd integration-tests/jail
+  - ./run.sh
```

Comment:
could you take out things from `run.sh` and make an explicit cleanup step like in: https://github.com/crypto-com/chain/blob/master/.drone.yml#L36

+ trigger only on bors branches (you can test with `bors try`): https://github.com/crypto-com/chain/blob/master/.drone.yml#L46 to save time

Reply:
ok, i'll check it